pkg/vexhub/vexhub_test.go

```diff
@@ -41,12 +42,14 @@ func TestGenerateIndex(t *testing.T) {
 				return os.WriteFile(manifestPath, data, 0644)
 			},
 			wantErr: require.NoError,
-			wantIndex: `{
-				"version": 1,
-				"packages": [
-					{ "id" : "package1", "location": "package1/source1" }
-				]
-			}`,
+			wantIndex: func(updatedAt time.Time) string {
+				return `{
+					"updated_at": "` + updatedAt.Format(time.RFC3339Nano) + `",
+					"packages": [
+						{ "id" : "package1", "location": "package1/source1" }
+					]
+				}`
+			},
```

Comment:
We don't need a function anymore.

```suggestion
			wantIndex: , `{
				"updated_at": "2025-05-20T17:16:15.123456789Z",
				"packages": [
					{ "id" : "package1", "location": "package1/source1" }
				]
			}`
```

Reply:
Thanks